From 05b1fe7a297c994d537f85bd9d71242ecb9f9c7b Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:03:59 -0400
Subject: [powerpc] idle: Convert use of typedef ctl_table to struct ctl_table

Message-id: <1410545655-205645-11-git-send-email-dzickus@redhat.com>
Patchwork-id: 94124
O-Subject: [RHEL7 PATCH 010/626] powerpc/idle: Convert use of typedef ctl_table to struct ctl_table
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit cc293bf7a9aa6fad68d107c79705732bd2fc57e3
Author: Joe Perches <joe@perches.com>
Date:   Thu Jun 13 19:37:30 2013 -0700

    powerpc/idle: Convert use of typedef ctl_table to struct ctl_table

    This typedef is unnecessary and should just be removed.

    Signed-off-by: Joe Perches <joe@perches.com>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/kernel/idle.c b/arch/powerpc/kernel/idle.c
index 939ea7e..d7216c9 100644
--- a/arch/powerpc/kernel/idle.c
+++ b/arch/powerpc/kernel/idle.c
@@ -85,7 +85,7 @@ int powersave_nap;
 /*
  * Register the sysctl to set/clear powersave_nap.
  */
-static ctl_table powersave_nap_ctl_table[]={
+static struct ctl_table powersave_nap_ctl_table[] = {
  {
   .procname = "powersave-nap",
   .data  = &powersave_nap,
@@ -95,7 +95,7 @@ static ctl_table powersave_nap_ctl_table[]={
  },
  {}
 };
-static ctl_table powersave_nap_sysctl_root[] = {
+static struct ctl_table powersave_nap_sysctl_root[] = {
  {
   .procname = "kernel",
   .mode  = 0555,
-- 
1.7.1